controllers/ai.controller.js

```diff
@@ -10,50 +11,83 @@ const url = `mongodb+srv://${process.env.DB_USER}:${process.env.DB_PASS}@${proce
 
 // create account
 exports.ai_newSync = function (req, res) {
-    res.render("pages/sync-module", {
+	res.render("pages/sync-module", {
 		title: "Sync new module",
-    });
+	});
 };
 
 // create
 exports.ai_create = function (req, res) {
-    mongoose.connect(url,  { useNewUrlParser: true }).then(
-        () => {
-            // construct model
+	mongoose.connect(url, {
+		useNewUrlParser: true
+	}).then(
+		() => {
+			// construct model
 			let AiSchema = mongoose.model("Ai", schema);
 
 			// format date so db is happy
-			let formatDate = moment(req.body.contructionDate).format("YYYY-DD-MM");
-
-            // create new instance of user
-            let ai = new AiSchema({
-                serialNr : req.body.serialNr,
-                password: req.body.password,
-                instanceName: req.body.instanceName,
-                brandName: req.body.brandName,
+			let formatDate = moment(req.body.constructionDate).format("YYYY-DD-MM");
+
+			// create new instance of user
+			let ai = new AiSchema({
+				serialNr: req.body.serialNr,
+				password: req.body.password,
+				instanceName: req.body.instanceName,
+				brandName: req.body.brandName,
 				constructionDate: new Date(formatDate),
 				gender: req.body.gender,
-				properties : {
+				properties: {
 					intelligence: req.body.intelligence,
 					environment: req.body.environment,
 					shape: req.body.shape,
 					abilities: req.body.abilities
 				},
 			});
 
-            // save user
-            ai.save(function (err) {
+			// save user
+			ai.save(function (err) {
 				if (err) {
 					console.log(err);
-				}
-                else {
+				} else {
 					orbit.createOrbit(ai);
-                    res.send(`User created: \n ${ai}`);
-                }
+					res.send(`User created: \n ${ai}`);
+				}
 			});
 
 			//return res.redirect("myAI");
-        },
-        err => { console.log(err); }
-    );
-};
\ No newline at end of file
+		},
+		err => {
+			console.log(err);
+		}
+	);
+};
+
+// update account
+exports.ai_update = function (req, res) {
+	utils.findAi(req.session.passport.user).then(function (ai) {
+		res.render("pages/update", {
+			title: "Update my AI",
```

Comment:
App name should be in the title like this: 
title : `${process.env.APP_NAME} - Orbit`,